.storybook/blocks/ComponentDetails.jsx

```diff
@@ -291,26 +429,28 @@ export const ComponentDetails = () => {
 
 	return (
 		<ResetWrapper>
-			{ !isLoading ?
-				<DList className="docblock-metadata sb-unstyled">
-					{ isDeprecated
-						?	<>
-								<DTerm key={'status'}>Status:</DTerm>
-								<DDefinition key={'status-data'}>Deprecated</DDefinition>
-							</>
-						: ""
-					}
-					{ showLocalVersion
-						?	<>
-								<DTerm key={'version-label'}>Local version:</DTerm>
-								<DDefinition key={'version'}><VersionDetails tag={"local"} data={allVersions && allVersions.find(([tag]) => tag === "local")?.[1]} isDeprecated={isDeprecated} /></DDefinition>
+			{ !isLoading ? <>
+					<DList className="docblock-metadata sb-unstyled">
+						{ isDeprecated
+							?	<>
+									<DTerm key={'status'}>Status:</DTerm>
+									<DDefinition key={'status-data'}>Deprecated</DDefinition>
+								</>
+							: ""
+						}
+						{ showLocalVersion
+							?	<>
+									<DTerm key={'version-label'}>Local version:</DTerm>
+									<DDefinition key={'version'}><VersionDetails tag={"local"} data={allVersions && allVersions.find(([tag]) => tag === "local")?.[1]} isDeprecated={isDeprecated} /></DDefinition>
+								</>
+							: <>
+								<DTerm key={'version-label'}>Latest version:</DTerm>
+								<DDefinition key={'version'}><VersionDetails tag={"latest"} data={allVersions && allVersions.find(([tag]) => tag === "latest")?.[1]} isDeprecated={isDeprecated} skipLink={true} /></DDefinition>
 							</>
-						: <>
-							<DTerm key={'version-label'}>Latest version:</DTerm>
-							<DDefinition key={'version'}><VersionDetails tag={"latest"} data={allVersions && allVersions.find(([tag]) => tag === "latest")?.[1]} isDeprecated={isDeprecated} skipLink={true} /></DDefinition>
-						</>
-					}
-				</DList>
+						}
+					</DList>
+					<ResourceListDetails packageName={packageName} spectrumData={spectrumData} rootClassName={rootClassName}/>
```

Comment:
One very minor thing that I'm noticing is that the GitHub links for the deprecated components don't work as intended. If it's not worthwhile even worrying about that and that was a conversation that already happened somewhere, that's fine!

If not, there are probably several ways we could handle this, and at this point you would probably have the best judgment on which to use, but some thoughts that come to my mind are:
* Only display the npm link for the deprecated components - either pass down `isDeprecated` to `ResourceListDetails` (I don't love this idea), or maybe only conditionally render the GitHub `ResourceLinkContent` if `spectrumData.length !== 0`?
* Don't render `ResourceListDetails` for the deprecated components at all? So either only render `ResourceListDetails` if `isDeprecated` is false, or return early if `spectrumData.length === 0` (`if (!packageName || spectrumData.length === 0) return;`)

What do you think?

Reply:
Going to note now that I've gone through the rest of the components that Action menu has no `spectrumData` (there's no `spectrum` key in the `package.json`), is that deliberate? If so that would have some unintended side effects with these approaches I just mentioned that involve checking the length of `spectrumData`

Reply:
Correct- it was deliberate to _not_ have a `spectrum` key for action menu.